proto/autokitteh/apply/v1/svc.proto

```diff
@@ -11,6 +11,7 @@ message ApplyRequest {
 
 message ApplyResponse {
   repeated string logs = 1;
+  repeated string project_ids = 2;
```

Comment:
why not returning all ids? do we want to return only stuff that created or also modified? or maybe just everything?

Reply:
Also, don't we support only a single project now in our manifest schema?